core/notifications/src/graphql/convert.rs

```diff
@@ -10,10 +10,11 @@ impl From<history::StatefulNotification> for types::StatefulNotification {
         Self {
             deep_link: notification.deep_link().map(|d| d.to_link_string()),
             id: ID(notification.id.to_string()),
-            title: notification.message.title,
-            body: notification.message.body,
+            title: notification.message.title.clone(),
+            body: notification.message.body.clone(),
             created_at: types::Timestamp::from(created_at),
             acknowledged_at: acknowledeg_at.map(types::Timestamp::from),
+            add_to_bulletin: notification.add_to_bulletin(),
```

Comment:
move these above line 13 - then you can de-structure without needing to clone. Best to avoid clone where possible.
Before you start moving attributes you can call fns.